packages/api-derive/src/staking/query.ts

```diff
@@ -84,17 +99,37 @@ function getStashInfo (api: DeriveApi, stashIds: AccountId[], activeEra: EraInde
       : of(stashIds.map(() => emptyExpo)),
     withExposureMeta
       ? combineLatest(stashIds.map((s) => api.query.staking.erasStakersOverview(activeEra, s)))
-      : of(stashIds.map(() => emptyExpoMeta))
+      : of(stashIds.map(() => emptyExpoMeta)),
+    withClaimedRewardsEras
+      ? combineLatest(stashIds.map((s) =>
+        combineLatest(
+          eras.map((e) => api.query.staking.claimedRewards(e, s))
+        ))
+      ).pipe(
+        map((r) => {
+          return r.map((stashClaimedEras) => {
+            // stashClaimedEras length will match the length of eras
+            return stashClaimedEras.map((claimedReward, idx) => {
+              if (claimedReward.length) {
```

Comment:
This isn't completely accurate and would need to be verified against how many actual pages there are.

Reply:
Updated this to verify against the overview pages